jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectorViewPanels.kt

```diff
@@ -377,6 +382,14 @@ class EnvironmentDetailsPanel(private val context: CawsSettings, lifetime: Lifet
                                 linkedBranchCombo.proposeModelUpdate { model ->
                                     projectCombo.selected()?.let { project ->
                                         linkedRepoCombo.selected()?.let { repo ->
+                                            val is3P = isRepo3P(project, repo.name, client)
+                                            if (is3P) {
+                                                branchOptions.visible(false)
+                                                newBranch.visible(false)
+                                            } else {
+                                                branchOptions.visible(true)
+                                                existingBranchOption.component.isSelected = true
```

Comment:
don't we need to make sure that `existingBranchOption` is selected if 3p?

Reply:
Actually, we are removing the whole radioButton options if 3p selected, only showing existing branch content.

Reply:
if the user previously selected "Create new branch", we need to flip the option back